pipeline/inputs/kafka.md

```diff
@@ -0,0 +1,75 @@
+# Kafka
+
+Kafka input plugin allows to subscribe to one or more topics and collect messages from an [Apache Kafka](https://kafka.apache.org/) service. This plugin use the official [librdkafka C library](https://github.com/edenhill/librdkafka) \(built-in dependency\)
+
+## Configuration Parameters
+
+| Key | Description | default |
+| :--- | :--- | :--- |
+| brokers | Single of multiple list of Kafka Brokers, e.g: 192.168.1.3:9092, 192.168.1.4:9092. |  |
```

Comment:
typo `of` --> `or`

Reply:
Is it comma separated, what about spaces?

Reply:
AFAIK only comma-separated values are accepted.